rest-tests/src/test/resources/features/operator-deployment.feature

```diff
@@ -319,3 +322,162 @@ Feature: Operator Deployment
       And check that extension "set-sqs-group-id-extension" exists
       And verify that integration with name "amq-amq-backup" exists
 
+  @ENTESB-12114
+  @ENTESB-12846
+  @operator-backup-restore
+  @operator-backup-restore-manual
+  Scenario: Syndesis Operator - Backup and Restore manual procedure
```

Comment:
Just a question: could these three scenarios be merged into a Scenario Outline with Examples?

Reply:
good point